src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -355,8 +358,12 @@ def test_WINNF_FT_S_FAD_1(self, config_filename):
           self.assertEqual(ppa_record['id'].split("/")[1], 'ppa')
           self.assertEqual(ppa_record['id'].split("/")[2], self._sas._sas_admin_id)
           del ppa_record['id']
+          # TODO : to update the following check according when PAL grants are supported in this test case
+          if 'cbsdReferenceId' in ppa_record['ppaInfo']:
+            self.assertFalse(ppa_record['ppaInfo']['cbsdReferenceId'])
```

Comment:
I'm not quite sure what the logic is here, but I guess the code will be removed as part of the TODO anyway.